packages/api/src/platforms/vtex/resolvers/shippingSLA.ts

```diff
@@ -0,0 +1,64 @@
+import type { Resolver } from '..'
+
+type Unit = 'bd' | 'd' | 'h' | 'm'
+const units = ['bd', 'd', 'h', 'm'] as const
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isUnit = (x: any): x is Unit => units.includes(x)
+
+const friendlyEstimates: Record<string, Record<string, string>> = {
```

Comment:
I know this came from the `shipping-estimate-translator`, but how about we call the formatted shipping estimates `localizedEstimates`? I think it conveys more clearly what we're actually returning

Reply:
Nice! addressed [here](https://github.com/vtex/faststore/pull/1440/commits/9c3675835d343adf50825925de9b0027d79b89bf).